setup/webmail.sh

```diff
@@ -122,7 +125,19 @@ cat > $RCM_CONFIG <<EOF;
 \$config['support_url'] = 'https://mailinabox.email/';
 \$config['product_name'] = '$PRIMARY_HOSTNAME Webmail';
 \$config['des_key'] = '$SECRET_KEY';
-\$config['plugins'] = array('html5_notifier', 'archive', 'zipdownload', 'password', 'managesieve', 'jqueryui', 'persistent_login', 'carddav');
+EOF
+
+if [ "${DISABLE_NEXTCLOUD}" == "0" ]; then
+	cat >> $RCM_CONFIG <<EOF;
+	\$config['plugins'] = array('html5_notifier', 'archive', 'zipdownload', 'password', 'managesieve', 'jqueryui', 'persistent_login');
+	EOF
+else
+	cat >> $RCM_CONFIG <<EOF;
+	\$config['plugins'] = array('html5_notifier', 'archive', 'zipdownload', 'password', 'managesieve', 'jqueryui', 'persistent_login', 'carddav');
```

Comment:
Can we reduce this code duplication even further?

PLUGINS="'html_notifier', ..."
if [ $DISABLE_NEXTCLOUD ]; then PLUGINS="$PLUGINS, 'carddav'"; fi

Sorry for the pseudo-code but I'm on my iPad again and I find it hard to type on.  Hopefully you get the gist!

Reply:
Can you elaborate further? Thanks.

Reply:
Never mind, figured it out.
Will fix it ASAP.

Reply:
I think something like this should mean we only have to keep one copy of the roundcube plugins we always install and then only add carddav if we are adding Nextcloud:

RC_PLUGINS="'html5_notifier', 'archive', 'zipdownload', 'password', 'managesieve', 'jqueryui', 'persistent_login'"
if [ "${DISABLE_NEXTCLOUD}" != "0" ]; then RC_PLUGINS="${RC_PLUGINS}, 'carddav'"; fi
cat >> $RCM_CONFIG <<EOF;
\$config['plugins'] = array(${RC_PLUGINS});
EOF

Reply:
FIxed, thanks again.